src/components/AddContent/AddContent.tsx

```diff
@@ -327,14 +327,9 @@ const AddContent = ({ isLoading }: Props) => {
           ouiaId='add_custom_repository'
           help={
             <Popover
-              headerContent={<div>Help Popover</div>}
-              bodyContent={
-                <div>
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam id feugiat augue,
-                  nec fringilla turpis.
-                </div>
-              }
-              footerContent='Popover Footer'
+              headerContent={<div>Add a custom repository</div>}
+              bodyContent={<div>Use this form to enter the values for a new repository.</div>}
+             
```

Comment:
there is an empty space here that irks me.. kill it with fire!

Reply:
Done, thanks